DevCycle.SDK.Server.Cloud/Api/DVCCloudClient.cs

```diff
@@ -195,5 +195,14 @@ public override void Dispose()
         {
             ((IDisposable) apiClient).Dispose();
         }
+        
+        /**
+         * Rebuild the logging of the client and all subcomponents
+         */
+        public override void UpdateLogging(ILoggerFactory loggerFactory) {
```

Comment:
I would setup another function call here that could override the logger straight through - with no logger factory init - incase there's a weird logging setup that's custom.

Reply:
A single logger instance would be much cleaner to work through as well. 
I'd recommend that over this for sure